libs/eavmlib/src/esp.erl

```diff
@@ -279,6 +284,34 @@ deep_sleep() ->
 deep_sleep(_SleepMS) ->
     erlang:nif_error(undefined).
 
+%%-----------------------------------------------------------------------------
+%% @param   Source the device that will be mounted
+%% @param   Target the path where the filesystem will be mounted
+%% @param   FS the filesystem, only fat is supported now
+%% @param   Opts
+%% @returns either a tuple having `ok' and the mounted fs resource, or an error tuple
+%% @doc     Mount a filesystem, and return a resource that can be used later for unmounting it
+%% @end
+%%-----------------------------------------------------------------------------
+-spec mount(
+    Source :: unicode:chardata(),
+    Target :: unicode:chardata(),
+    FS :: fat,
+    Opts :: proplists:proplist() | #{atom() => term()}
+) -> {ok, mounted_fs()} | {error, Reason}.
```

Comment:
Not sure if this is related to the CI build error, but you might want to specify `Reason :: term()` here, and again below.

Reply:
that's the quick typical lunch break change mistake !